utils/swift_build_support/swift_build_support/products/skstresstester.py

```diff
@@ -21,3 +25,62 @@ def product_source_name(cls):
         The name of the source code directory of this product.
         """
         return "swift-stress-tester"
+
+    def run_build_script_helper(self, action, additional_params=[]):
+        script_path = os.path.join(
+            self.source_dir, 'build-script-helper.py')
+        toolchain_path = self.args.install_destdir
+        if platform.system() == 'Darwin':
+            # The prefix is an absolute path, so concatenate without os.path.
+            toolchain_path += \
+                targets.darwin_toolchain_prefix(self.args.install_prefix)
+        
+        toolchain_bin = os.path.join(toolchain_path, 'usr', 'bin')
+        configuration = 'debug' if self.args.build_variant == 'Debug' else \
+            'release'
+        
+        helper_cmd = [
+            script_path,
+            action,
+            '--package-dir', 'SourceKitStressTester',
+            '--build-dir', self.build_dir,
+            '--swiftc-exec', os.path.join(toolchain_bin, 'swiftc'),
+            '--swift-build-exec', os.path.join(toolchain_bin, 'swift-build'),
+            '--swift-test-exec', os.path.join(toolchain_bin, 'swift-test'),
+            '--syntax-parser-header-dir', \
+                os.path.join(toolchain_path, 'usr', 'lib', 'swift'),
```

Comment:
I don't see SwiftSyntaxParser.h in the toolchain at all... but I'm also not sure why we need this path.  It seems to be optional, could we just drop it?

Reply:
I have it installed in the toolchain, both the one I built locally as well as the open source one. I lives in `/usr/lib/swift/_InternalSwiftSyntaxParser`

Reply:
Ah, I was looking at an older toolchain by mistake.   I talked to @nathawes who said we shouldn't really need this path when building from build-script. It's used for local development when you want to build the stress tester and swift syntax together, but not build all of swift.

Reply:
Do you mind if I take care of this when migrating the stress tester’s build script to use the toolchain?

Reply:
SGTM